programs/marginfi/src/state/marginfi_group.rs

```diff
@@ -1402,8 +1445,24 @@ impl BankConfig {
         self.borrow_limit != u64::MAX
     }
 
-    pub fn validate_oracle_setup(&self, ais: &[AccountInfo]) -> MarginfiResult {
-        OraclePriceFeedAdapter::validate_bank_config(self, ais)?;
+    /// * lst_mint, stake_pool, sol_pool - required only if configuring `OracleSetup::StakedWithPythPush`
+    pub fn validate_oracle_setup(
+        &self,
+        ais: &[AccountInfo],
+        lst_mint: Option<Pubkey>,
+        stake_pool: Option<Pubkey>,
+        sol_pool: Option<Pubkey>,
+    ) -> MarginfiResult {
+        OraclePriceFeedAdapter::validate_bank_config(self, ais, lst_mint, stake_pool, sol_pool)?;
+        Ok(())
+    }
```

Comment:
Should we maybe have `validate_bank_config` and `validate_stake_bank_config` to avoid the complex parameter permutations and manage complexity of the function internally, or use a single option with a tuple.